### PR TITLE
Fix dma resource (Foxeer F722V2 target config)

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: #480

### CODE REVIEW

**Comment by haslinghuis** [comment]
@Johnsnow-foxeer Is this ok? I made the change you had requested on your behalf.

**Comment by Johnsnow-foxeer** [comment]
thank you, testing now

**Comment by Johnsnow-foxeer** [comment]
anyone can review?

**Comment by DusKing1** [comment]
@Johnsnow-foxeer need your unified target administrator's review, in this case, I think should be @nyway

**Comment by Johnsnow-foxeer** [comment]
OK, can you help? @nyway

**Comment by Johnsnow-foxeer** [comment]
We have some FCs need to ship out as soon as possible, because the US IO racing is coming, some customers are very hurried to get the FC, thank you

**Comment by DusKing1** [comment]
@Johnsnow-foxeer As I clarified above, you yourself are the administrator. And I can do nothing on your shipping issue.

**Comment by Johnsnow-foxeer** [comment]
I am sorry, I am not familiar with the procedure, how can I do now?

**Review by Johnsnow-foxeer — APPROVED** [review]

**Comment by Johnsnow-foxeer** [comment]
I approved the change, what is next?

**Comment by haslinghuis** [comment]
Somehow @Johnsnow-foxeer does not have approval rights:

image

**Comment by Johnsnow-foxeer** [comment]
Yes, seems need @DusKing1 review

**Comment by haslinghuis** [comment]
@Johnsnow-foxeer No, as administrator you should have this right as you should have made this change as maintainer of Foxeer.

@mikeller @McGiverGim @etracer65 IDK what's wrong here.

**Review by Johnsnow-foxeer — APPROVED** [review]

**Comment by Johnsnow-foxeer** [comment]
I think I know the issue, I am not the administrator, this is the V2 version of our FC, but the V1 version is from another engineer, that engineer resigned already, can you change me to the administrator? or I need to connect with previous engineer?

**Comment by Johnsnow-foxeer** [comment]
@DusKing1 I think I am not the administrator, seems I have no rights to approve

**Comment by DusKing1** [comment]
@haslinghuis yeah somehow it seems that I don't have access to the RUSH either.

**Comment by Johnsnow-foxeer** [comment]
Now, what can I do to get this passed..

**Comment by Johnsnow-foxeer** [comment]
Please help

File: configs/default/FOXE-FOXEERF722V2.config
Comment (mikeller) [inline configs/default/FOXE-FOXEERF722V2.config:71]: This is incorrect now, making it very confusing.

Comment (Johnsnow-foxeer) [reply configs/default/FOXE-FOXEERF722V2.config:71]: When flash the new firmware, and change to Dshot, the problem happened, then I have to import the config file manually, the previous config file is no problem.

**Review by Johnsnow-foxeer — APPROVED** [review]

File: configs/default/FOXE-FOXEERF722V2.config
Comment (DusKing1) [reply configs/default/FOXE-FOXEERF722V2.config:71]: @Johnsnow-foxeer mikeller was not refering this` dma ADC 3 0` line, instead it was the # ADC 3 : DMA2 Stream 1 Channel 2 does not correspond to its actual assignment. You need to also change this according to a real FC's output.

Comment (DusKing1) [reply configs/default/FOXE-FOXEERF722V2.config:71]: @haslinghuis should be: `# ADC 3: DMA2 Stream 0 Channel 2`

**Review by DusKing1 — CHANGES_REQUESTED** [review]

File: configs/default/FOXE-FOXEERF722V2.config
Comment (DieHertz) [reply configs/default/FOXE-FOXEERF722V2.config]: @DusKing1 Harvey asked me to help, was the out-of-sync commentary all that's wrong with this PR?

**Comment by DusKing1** [comment]
@DieHertz thanks for your new commit, now I think this should fix their problem

**Review by DusKing1 — APPROVED** [review]

File: configs/default/FOXE-FOXEERF722V2.config
Comment (mikeller) [reply configs/default/FOXE-FOXEERF722V2.config]: @DieHertz: No, what is wrong is that this is simply reverting to a version that is not optimised for SPI DMA that will lead to poor performance for this target. The other problem is the fact that 'Harvey' (@Johnsnow-foxeer) seems to be hellbound to kill progress and refuses to properly test the actual fix (#483) or provide an explanation of what the problem is.

Comment (DusKing1) [reply configs/default/FOXE-FOXEERF722V2.config]: @mikeller I tried to reproduce what might happened to his target and this is the detail that I can provide: https://github.com/betaflight/unified-targets/issues/493.

However I don't understand their engineer's "problem description " so I'm not sure what they actually did.

Comment (DieHertz) [reply configs/default/FOXE-FOXEERF722V2.config]: I see, did not notice the other PR and discussion, will look into it

Comment (mikeller) [reply configs/default/FOXE-FOXEERF722V2.config]: @DusKing1: Added a comment and a video to #493 - as I'd stated before, what you say gives you a lockup is working fine for me when testing.

**Review by mikeller — APPROVED** [review]
Mergeing this until the problem described in #493 has been fixed.

**Pull request merged**
